x-pack/plugins/security_solution/public/resolver/store/middleware/related_events_fetcher.ts

```diff
@@ -29,20 +29,42 @@ export function RelatedEventsFetcher(
     // Update this each time before fetching data (or even if we don't fetch data) so that subsequent actions that call this (concurrently) will have up to date info.
     last = newParams;
 
-    // If the panel view params have changed and the current panel view is either `nodeEventsOfType` or `eventDetail`, then fetch the related events for that nodeID.
-    if (
-      !isEqual(newParams, oldParams) &&
-      (newParams.panelView === 'nodeEventsOfType' || newParams.panelView === 'eventDetail')
-    ) {
-      const nodeID = newParams.panelParameters.nodeID;
+    // If the panel view params have changed and the current panel view is either `nodeEventsInCategory` or `eventDetail`, then fetch the related events for that nodeID.
+    if (!isEqual(newParams, oldParams)) {
```

Comment:
ℹ️ ROBERT AUSTIN USING LODASH FOLKS

Reply:
This is the year to experience traumatic and unexpected things right?